tilecloud/store/queue.py

```diff
@@ -0,0 +1,28 @@
+import base64
+import json
+
+from tilecloud import Tile, TileCoord
+
+
+def encode_message(tile):
+    sqs_message = {
+        'z': tile.tilecoord.z,
+        'x': tile.tilecoord.x,
+        'y': tile.tilecoord.y,
+        'n': tile.tilecoord.n,
+        'metadata': tile.metadata
+    }
+    if 'sqs_message' in sqs_message['metadata']:
+        del sqs_message['metadata']['sqs_message']
+
+    return base64.b64encode(json.dumps(sqs_message).encode('utf-8')).decode('utf-8')
```

Comment:
Why those SQS messages in generic queue file?

Reply:
Incomplete refactoring. Fixed...